src/fixate/drivers/dmm/__init__.py

```diff
@@ -11,16 +11,22 @@
 import fixate.drivers
 from fixate.drivers.dmm.helper import DMM
 from fixate.drivers.dmm.fluke_8846a import Fluke8846A
+from fixate.drivers.dmm.keithley_6500 import Keithley6500
 from fixate.config import find_instrument_by_id
 
 
 def open() -> DMM:
-    instrument = find_instrument_by_id(Fluke8846A.REGEX_ID)
-    if instrument is not None:
-        # we've found a connected instrument so open and return it
-        rm = pyvisa.ResourceManager()
-        # open_resource could raise visa.VisaIOError?
-        driver = Fluke8846A(rm.open_resource(instrument.address))
-        fixate.drivers.log_instrument_open(driver)
-        return driver
+
+    for DMM in [Fluke8846A, Keithley6500]:
+        instrument = find_instrument_by_id(DMM.REGEX_ID)
+        if instrument is not None:
+            # Attempt to open instrument in config file
+            rm = pyvisa.ResourceManager()
+            try:
+                driver = DMM(rm.open_resource(instrument.address))
+                fixate.drivers.log_instrument_open(driver)
+                return driver
+            except pyvisa.errors.VisaIOError:
+                # Tried to open a DMM that is in the config, but not physically connected.
+                pass
```

Comment:
There are two different errors here that we probably want to differentiate: an instrument exists in the config but can't be opened (pyvisa error), or there is no instrument configured on the station (InstrumentNotFoundError).

In the visa except - should at least 'break' instead of pass as the consensus was not allowing multiple devices of the same type to be configured/tried.

I started PR #161 to better handle the instrument open errors - if/when merged could consolidate

Reply:
I think I'll get PR #161 merged, then we can look at what changes need to be made here taking the changes there into account.

Reply:
If this is otherwise ready to merge, you could add a log in the except clause, so the issues is not completely silently ignored. Then #161 can tackle the wider issue.

Reply:
You can't tell from the snippet above, but in the wider context the VisaIOError here is being used for flow control (not a huge fan of using exceptions for flow control, but I'm assuming open_resource doesn't just return None if it couldn't find the instrument) for the loop through the DMM list. If nothing found, InstrumentNotFoundError will still be raised, so I think this instance is one where silent ignorance is ok.

As I type this, I decided I do agree with adding a log message (and possibly display a warning message) that there is an instrument in the config that's not connected (is my memory correct in that we don't really want instruments hanging around in the config if they're not connected?)

Reply:
My assumption when first writing that was that the config can have both the Keithley and Fluke instruments set up. I have since changed that to assume only one instrument is present in the config (the way it should be?).